src/components/FormField/component.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {
+  Form, Input
+} from 'antd'
+import PropTypes from 'prop-types'
+
+const FormField = ({
+  placeholder, validateStatus, help, field, prefix: PrefixComponent, ...inputProps
```

Comment:
why not to name `PrefixComponent` just `Prefix`?

Reply:
also not sure about `validateStatus`. lets change it to `validationStatus`

Reply:
@VladWithCookies  Props `help` & `validateStatus` are from `antd` lib https://ant.design/components/form/#Form.Item

Reply:
> why not to name PrefixComponent just Prefix?

To emphasize the type of props

Reply:
@kosovych, i am agree with you about the antd design stuff. but i am not sure about `PrefixComponent`. because you use `PropTypes` package which already gives you some tool for type anotation, so you don't need to anotate property type by using some prefixes or suffixes

Reply:
fixed